modules/tensor_mechanics/src/actions/DynamicTensorMechanicsAction.C

```diff
@@ -0,0 +1,116 @@
+/****************************************************************/
```

Comment:
This file is more or less a copy-paste of ```TensorMechanicsAction``` please inherit from that class! Whenever you find yourself in a situation where you are copy-pasting code take a step back and try to come up with a better way.